src/graph_export/graph_cli_parser.cpp

```diff
@@ -0,0 +1,158 @@
+            cxxopts::value<std::string>()->default_value(""),
+            "TASK")
+            ("max_num_seqs",
+                "The maximum number of sequences that can be processed together. Default 256.",
+                cxxopts::value<std::string>()->default_value("256"),
+                "MAX_NUM_SEQS")
+            ("pipeline_type",
+                "Type of the pipeline to be used: Choices LM, LM_CB, VLM, VLM_CB, AUTO. AUTO is used by default.",
+                cxxopts::value<std::string>()->default_value("AUTO"),
+                "PIPELINE_TYPE")
+            ("graph_target_device",
+                "CPU, GPU, NPU or HETERO, default is CPU.",
+                cxxopts::value<std::string>()->default_value("CPU"),
+                "GRAPH_TARGET_DEVICE")
+            ("enable_prefix_caching",
+                "This algorithm is used to cache the prompt tokens.",
+                cxxopts::value<std::string>()->default_value("true"),
+                "ENABLE_PREFIX_CACHING")
+            ("max_num_batched_tokens",
+                "empty or integer. The maximum number of tokens that can be batched together.",
+                cxxopts::value<std::string>()->default_value(""),
+                "MAX_NUM_BATCHED_TOKENS")
+            ("cache_size",
+                "cache size in GB, default is 10.",
+                cxxopts::value<std::string>()->default_value("10"),
+                "CACHE_SIZE")
+            ("draft_source_model",
+                "HF model name or path to the local folder with PyTorch or OpenVINO draft model.",
+                cxxopts::value<std::string>()->default_value(""),
+                "DRAFT_SOURCE_MODEL")
+            ("dynamic_split_fuse",
+                "Dynamic split fuse algorithm enabled. Default true.",
+                cxxopts::value<std::string>()->default_value("false"),
+                "DYNAMIC_SPLIT_FUSE");
+
+        options->add_options("plugin config")
+            ("max_prompt_len",
+                "Sets NPU specific property for maximum number of tokens in the prompt.",
+                cxxopts::value<std::string>()->default_value(""),
+                "MAX_PROMPT_LEN")
+            ("kv_cache_precision",
+                "u8 or empty (model default). Reduced kv cache precision to u8 lowers the cache size consumption.",
+                cxxopts::value<std::string>()->default_value(""),
+                "KV_CACHE_PRECISION");
+}
+
+void GraphCLIParser::printHelp() {
+    if (!this->options) {
+        this->createOptions();
+    }
+    std::cout << options->help({"general options", "plugin config"}) << std::endl;
+}
+
+void GraphCLIParser::parse(const std::vector<std::string>& unmatchedOptions) {
+    try {
+        if (!this->options) {
+            this->createOptions();
+        }
+        std::vector<const char*> cStrArray;
+        cStrArray.reserve(unmatchedOptions.size()+1);
+        cStrArray.push_back("ovms graph");
```

Comment:
Is it actually used? If not maybe be more explicit with "UNUSED"?

Reply:
This is just a header. Might be useful when exceptions occur, so that we know what went wrong.

Reply:
Actually it is required if you use parse method.